keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClassicLoadBalancerHandlerTests.kt

```diff
@@ -63,17 +63,20 @@ internal class ClassicLoadBalancerHandlerTests : JUnit5Minutests {
   private val cloudDriverService = mockk<CloudDriverService>()
   private val cloudDriverCache = mockk<CloudDriverCache>()
   private val orcaService = mockk<OrcaService>()
-  private val clock = Clock.systemDefaultZone()
+  private val deliveryConfigRepository: InMemoryDeliveryConfigRepository = mockk() {
+    // we're just using this to get notifications
+    every { environmentFor(any()) } returns Environment("test")
+  }
   private val taskLauncher = TaskLauncher(
-    orcaService,
-    InMemoryDeliveryConfigRepository(clock)
+      orcaService,
+      deliveryConfigRepository
```

Comment:
What's going on with all the indentation changes in this file? Doesn't look right to me.

Reply:
I don't know! My spotless just did that :( do you have any idea how to fix it?

Reply:
That's weird. I just ran `gradle spotlessApply` and it didn't change anything. I wonder how we have different settings.

Reply:
intellij, you fool! Fixed.